Doc/library/typing.rst

```diff
@@ -1754,15 +1754,27 @@ without the dedicated syntax, as documented below.
          the constraints are evaluated only when the attribute is accessed, not when
          the type variable is created (see :ref:`lazy-evaluation`).
 
+   .. attribute:: __default__
+
+      The default value of the type variable, if any. This attribute is ``None``
```

Comment:
The docs here and for the other `__default__` attributes in `typing.rst` also need to be updated